comictaggerlib/comiccacher.py

```diff
@@ -346,13 +355,14 @@ def get_volume_issues_info(self, volume_id: int) -> list[CVIssuesResults]:
             for row in rows:
                 record = CVIssuesResults(
                     {
-                        "id": row[0],
-                        "name": row[1],
-                        "issue_number": row[2],
-                        "site_detail_url": row[3],
-                        "cover_date": row[4],
-                        "image": {"super_url": row[5], "thumb_url": row[6]},
-                        "description": row[7],
+                        "source_name": row[0],
+                        "id": row[1],
+                        "name": row[2],
+                        "issue_number": row[3],
+                        "site_detail_url": row[4],
+                        "cover_date": row[5],
+                        "image": {"super_url": row[6], "thumb_url": row[7]},
+                        "description": row[8],
```

Comment:
Same as above I recommend making source the last column, as it makes this a one line change.